packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -244,6 +245,9 @@ class PaginatedDataTable extends StatefulWidget {
   /// Defines the color of the arrow heads in the footer.
   final Color? arrowHeadColor;
 
+  ///Defines the background color of the table
```

Comment:
Please see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation and the next section.